### DISCUSSION TITLE
Google Sign in SPMP Desktop Application

### DISCUSSION THREAD

**Discussion by abhinavverma658** [sayaka-sh/spmp#343] [Q&A]
Can any one pls provide the solution for google sign-in in Desktop Spmp application?

**Comment by sayaka-sh** [comment]
I'm not sure what you're asking. Could you be more specific please?

**Comment by abhinavverma658** [comment]
![Screenshot 2024-07-13 114037](https://github.com/user-attachments/assets/19225b7c-2e89-494c-881c-36715af6c2b7)
Actually.. I'm talking about this ... I have tried all steps which are mentioned in this ... but all time it shows missing authorization and cookies. Any solution for google sign in in spmp desktop application.

**Comment by chaoyiuwu** [comment]
having the same issue! would really appreciate some more clarifications

**Comment by sayaka-sh** [comment]
Have either of you tried a recent debug build?
